### ISSUE TITLE
ionic google-maps 'Version should contain only numbers and dots'

### ISSUE THREAD

**Issue by Thierry-Santos** [ionic-team/ionic-native-google-maps#123]
**I'm submitting a ...**  (check one with "x")
- [ ] question
- [x] any problem or bug report
- [ ] feature request

**If you choose 'problem or bug report', please select OS:**  (check one with "x")
- [ ] **Android**
- [x] **iOS**
- [ ] **Browser**

**cordova information:**  (run `$> cordova plugin list`) 

```
com.telerik.plugins.nativepagetransitions 0.6.5 "Native Page Transitions"
cordova-plugin-add-swift-support 1.7.1 "AddSwiftSupport"
cordova-plugin-advanced-http 2.0.1 "Advanced HTTP plugin"
cordova-plugin-android-permissions 1.0.0 "Permissions"
cordova-plugin-camera 4.0.3 "Camera"
cordova-plugin-device 2.0.2 "Device"
cordova-plugin-file 6.0.1 "File"
cordova-plugin-geolocation 4.0.1 "Geolocation"
cordova-plugin-globalization 1.11.0 "Globalization"
cordova-plugin-googlemaps 2.4.6 "cordova-plugin-googlemaps"
cordova-plugin-ionic-keyboard 2.1.3 "cordova-plugin-ionic-keyboard"
cordova-plugin-ionic-webview 2.2.0 "cordova-plugin-ionic-webview"
cordova-plugin-nativegeocoder 3.1.2 "NativeGeocoder"
cordova-plugin-request-location-accuracy 2.2.3 "Request Location Accuracy"
cordova-plugin-secure-storage 2.6.8 "SecureStorage"
cordova-plugin-sms 1.0.5 "SMS"
cordova-plugin-sms-receive 1.0.1 "SMS Receive"
cordova-plugin-splashscreen 5.0.2 "Splashscreen"
cordova-plugin-statusbar 2.4.2 "StatusBar"
cordova-plugin-whitelist 1.3.3 "Whitelist"
cordova-sqlite-storage 2.5.0 "Cordova sqlite storage plugin"
cordova-support-google-services 1.1.0 "cordova-support-google-services"
phonegap-plugin-barcodescanner 8.0.0 "BarcodeScanner"
phonegap-plugin-multidex 1.0.0 "Multidex"
phonegap-plugin-push 2.2.3 "PushPlugin"
```

**If you use `@ionic-native/google-maps`, please tell the package.json (only `@ionic-native/core` and `@ionic-native/google-maps` are fine mostly)**

```
"@ionic-native/core": "~4.12.0",
"@ionic-native/google-maps": "^4.15.1",
```

**Current behavior:**
```
MBP-de-Thierry:app-ionic trbr$ ionic cordova platform add ios
> cordova platform add ios --save
--[cordova-plugin-googlemaps]------------------------
From version 2.4.5, the cordova-plugin-googlemaps uses CocoaPod.
No longer necessary com.googlemaps.ios plugin.
Automatic uninstalling com.googlemaps.ios plugin...
-----------------------------------------------------
Using cordova-fetch for cordova-ios@~4.5.4
Adding ios project...
Creating Cordova project for the iOS platform:
        Path: platforms/ios
        Package: (confidential).fcmtests.app
        Name: (confidential)
iOS project created with cordova-ios@4.5.5
Installing "com.telerik.plugins.nativepagetransitions" for ios
Installing "cordova-plugin-add-swift-support" for ios
Installing "cordova-plugin-advanced-http" for ios
Plugin dependency "cordova-plugin-file@6.0.1" already fetched, using that version.
Installing "cordova-plugin-file" for ios
Installing "cordova-plugin-android-permissions" for ios
Installing "cordova-plugin-camera" for ios
Installing "cordova-plugin-device" for ios
Dependent plugin "cordova-plugin-file" already installed on ios.
Installing "cordova-plugin-geolocation" for ios
Installing "cordova-plugin-globalization" for ios
Installing "cordova-plugin-googlemaps" for ios
--[cordova-plugin-googlemaps]------------------------
From version 2.4.5, the cordova-plugin-googlemaps uses CocoaPod.
No longer necessary com.googlemaps.ios plugin.
Automatic uninstalling com.googlemaps.ios plugin...
-----------------------------------------------------
Failed to install 'cordova-plugin-googlemaps': undefined
CordovaError: Promise rejected with non-error: 'Version should contain only numbers and dots'
    at cli.catch.err (/usr/local/lib/node_modules/cordova/bin/cordova:30:15)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:189:7)
[ERROR] An error occurred while running subprocess cordova.
```

**Screen capture or video record:**
<img width="667" alt="captura de tela 2018-11-07 as 15 55 59" src="https://user-images.githubusercontent.com/31314626/48150416-c822b880-e2a5-11e8-8abf-f0b9ab327f5b.png">

**Comment by wf9a5m75** [comment]
Please start from an empty project in order to reproduce your issue.

**Comment by Thierry-Santos** [comment]
Here it is, just: ionic cordova platform add ios.

https://github.com/Thierry-Santos/error-forceIonic

**Comment by wf9a5m75** [comment]
```
masashi-macgms:test masashi$ cd error-forceIonic/
masashi-macgms:error-forceIonic masashi$ npm i

> fsevents@1.2.4 install /Users/masashi/Documents/workspace2/test/error-forceIonic/node_modules/fsevents
> node install

[fsevents] Success: "/Users/masashi/Documents/workspace2/test/error-forceIonic/node_modules/fsevents/lib/binding/Release/node-v64-darwin-x64/fse.node" already installed
Pass --update-binary to reinstall or --build-from-source to recompile

> node-sass@4.9.0 install /Users/masashi/Documents/workspace2/test/error-forceIonic/node_modules/node-sass
> node scripts/install.js

Cached binary found at /Users/masashi/.npm/node-sass/4.9.0/darwin-x64-64_binding.node

> uglifyjs-webpack-plugin@0.4.6 postinstall /Users/masashi/Documents/workspace2/test/error-forceIonic/node_modules/uglifyjs-webpack-plugin
> node lib/post_install.js


> node-sass@4.9.0 postinstall /Users/masashi/Documents/workspace2/test/error-forceIonic/node_modules/node-sass
> node scripts/build.js

Binary found at /Users/masashi/Documents/workspace2/test/error-forceIonic/node_modules/node-sass/vendor/darwin-x64-64/binding.node
Testing binary
Binary is fine
added 917 packages from 694 contributors in 26.043s
masashi-macgms:error-forceIonic masashi$ ionic cordova platform add ios
✔ Creating ./www directory for you - done!
> cordova platform add ios --save
Using cordova-fetch for cordova-ios@~4.5.4
Adding ios project...
Creating Cordova project for the iOS platform:
	Path: platforms/ios
	Package: io.ionic.starter
	Name: MyApp
iOS project created with cordova-ios@4.5.5
Discovered plugin "cordova-plugin-whitelist" in config.xml. Adding it to the project
Installing "cordova-plugin-whitelist" for ios
Adding cordova-plugin-whitelist to package.json
Saved plugin info for "cordova-plugin-whitelist" to config.xml
Discovered plugin "cordova-plugin-statusbar" in config.xml. Adding it to the project
Installing "cordova-plugin-statusbar" for ios
Adding cordova-plugin-statusbar to package.json
Saved plugin info for "cordova-plugin-statusbar" to config.xml
Discovered plugin "cordova-plugin-device" in config.xml. Adding it to the project
Installing "cordova-plugin-device" for ios
Adding cordova-plugin-device to package.json
Saved plugin info for "cordova-plugin-device" to config.xml
Discovered plugin "cordova-plugin-splashscreen" in config.xml. Adding it to the project
Installing "cordova-plugin-splashscreen" for ios
Adding cordova-plugin-splashscreen to package.json
Saved plugin info for "cordova-plugin-splashscreen" to config.xml
Discovered plugin "cordova-plugin-ionic-webview" in config.xml. Adding it to the project
Installing "cordova-plugin-ionic-webview" for ios
Adding cordova-plugin-ionic-webview to package.json
Saved plugin info for "cordova-plugin-ionic-webview" to config.xml
Discovered plugin "cordova-plugin-ionic-keyboard" in config.xml. Adding it to the project
Installing "cordova-plugin-ionic-keyboard" for ios
Adding cordova-plugin-ionic-keyboard to package.json
Saved plugin info for "cordova-plugin-ionic-keyboard" to config.xml
Discovered plugin "cordova-plugin-geolocation" in config.xml. Adding it to the project
Installing "cordova-plugin-geolocation" for ios
Adding cordova-plugin-geolocation to package.json
Saved plugin info for "cordova-plugin-geolocation" to config.xml
Discovered plugin "cordova-plugin-nativegeocoder" in config.xml. Adding it to the project
Installing "cordova-plugin-nativegeocoder" for ios
Installing "cordova-plugin-add-swift-support" for ios
Adding cordova-plugin-nativegeocoder to package.json
Saved plugin info for "cordova-plugin-nativegeocoder" to config.xml
--save flag or autosave detected
Saving ios@~4.5.5 into config.xml file ...
Update IOS build setting SWIFT_OBJC_BRIDGING_HEADER to: "$(PROJECT_DIR)/$(PROJECT_NAME)/Bridging-Header.h" for build configuration Debug
Update IOS build setting SWIFT_OBJC_BRIDGING_HEADER to: "$(PROJECT_DIR)/$(PROJECT_NAME)/Bridging-Header.h" for build configuration Release
Importing NativeGeocoder-Bridging-Header.h into /Users/masashi/Documents/workspace2/test/error-forceIonic/platforms/ios/MyApp/Bridging-Header.h
Update IOS build setting ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES to: YES for build configuration Debug
Update SWIFT version to 3.0 Debug
Update IOS build setting SWIFT_OPTIMIZATION_LEVEL to: -Onone for build configuration Debug
Update IOS build setting ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES to: YES for build configuration Release
Update SWIFT version to 3.0 Release
masashi-macgms:error-forceIonic masashi$ 
```

**Comment by wf9a5m75** [comment]
Please try following things:

```
$> cordova plugin rm com.googlemaps.ios

$> cd plugins/

$> rm -rf com.googlemaps.ios

$> npm -g install cordova  // use the latest version

$> cordova plugin rm cordova-plugin-googlemaps

$> cordova plugin add cordova-plugin-googlemaps --variable API_KEY...
```

**Comment by wf9a5m75** [comment]
ping

**Comment by Thierry-Santos** [comment]
I solved using a old version of cocoapods, 1.2.1 thanks for your time.
